libs/features/interlinks.js

```diff
@@ -23,7 +23,7 @@ export function checkAndAddMatch(matches, contender, maxMatches) {
  * @param {string} path The location of the keywords file to be used for interlinks.
  * @param {number} limit The maximum amount of keywords to fetch from the file.  Default is 1000.
  */
-export default async function interlink(path, limit = 1000) {
+export default async function interlink(path = '/keywords.json', limit = 1000) {
```

Comment:
You should be using `${config.locale.contentRoot}/keywords.json`;

Reply:
I keep getting caught out by that configRoot don't I?